dali/operators/image/resize/experimental/resize_op_impl_cvcuda.h

```diff
@@ -210,54 +220,22 @@ class ResizeOpImplCvCuda : public ResizeBase<GPUBackend>::Impl {
 
   cvcuda::HQResize resize_op_{};
   nvcvop::NVCVOpWorkspace op_workspace_;
-  cvcuda::WorkspaceRequirements workspace_reqs_{};
+  std::array<cvcuda::WorkspaceRequirements, 2> workspace_reqs_{};
   std::vector<HQResizeRoiF> rois_;
   const TensorLayout sample_layout_ = (spatial_ndim == 2) ? "HWC" : "DHWC";
 
+  TensorList<GPUBackend> in_frames_;
+  TensorList<GPUBackend> out_frames_;
```

Comment:
It helped A LOT. For some reason The destructors of  theses were taking a lot of time. 
Anyway, I'm working on removing those auxiliary TensorLists completely, because the .ShareData and .Resize also take significant amount of time.